sharing/present.go

```diff
@@ -124,11 +124,13 @@ func (handler *Handler) showObject(ctx context.Context, w http.ResponseWriter, r
 	// on, otherwise we fall back to what wrapDefault was.
 	wrap := queryFlagLookup(q, "wrap",
 		!queryFlagLookup(q, "view", !pr.wrapDefault))
+	// flag for media previews.
+	previewFlag := queryFlagLookup(q, "preview", false)
 
 	if download {
 		w.Header().Set("Content-Disposition", "attachment")
 	}
-	if download || !wrap {
+	if previewFlag || download || !wrap {
```

Comment:
if all previewFlag controls is if we wrap, why don't we just use wrap=0 instead of preview?

Reply:
same as above